lms/templates/learner_dashboard/course_entitlement.underscore

```diff
@@ -0,0 +1,32 @@
+<div id="change-session-<%- entitlementUUID %>" class="message is-shown">
+    <div class="action-header">
+        <% if (currentSessionId) { %>
+            <%- gettext('Change to a different session or leave current session.')%>
```

Comment:
@HarryRein Please add "the": "...or leave the current session."

Reply:
Done